modules/core/test/test_intrin_utils.hpp

```diff
@@ -1573,22 +1573,22 @@ template<typename R> struct TheTest
         return *this;
     }
 
-#if 0
     TheTest & test_loadstore_fp16()
     {
+#if CV_SIMD_FP16
         printf("test_loadstore_fp16 ...\n");
         AlignedData<R> data;
         AlignedData<R> out;
 
         // check if addresses are aligned and unaligned respectively
-        EXPECT_EQ((size_t)0, (size_t)&data.a.d % VTraits<R>::max_nlanes);
-        EXPECT_NE((size_t)0, (size_t)&data.u.d % VTraits<R>::max_nlanes);
-        EXPECT_EQ((size_t)0, (size_t)&out.a.d % VTraits<R>::max_nlanes);
-        EXPECT_NE((size_t)0, (size_t)&out.u.d % VTraits<R>::max_nlanes);
+        EXPECT_EQ((size_t)0, (size_t)&data.a.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
+        EXPECT_NE((size_t)0, (size_t)&data.u.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
+        EXPECT_EQ((size_t)0, (size_t)&out.a.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
+        EXPECT_NE((size_t)0, (size_t)&out.u.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
```

Comment:
Story before these changes:

1. The dividend ` VTraits<R>::max_nlanes` was `CV_SIMD_WIDTH` in the previous change https://github.com/opencv/opencv/pull/22179.
2.  Macro `CV_SIMD_WIDTH` was replaced with `sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()` in the pull request, but somehow only these four lines were replaced with `VTraits<R>::max_nlanes`.
3. Tried printing values of these variables and macros, looks like the correct replacement should be `sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()` instead.
    ```
    VTraits<R>::max_nlanes = 4
    CV_SIMD_WIDTH=16
    sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()
    ```